Header header

sensor_msgs/Image  rgb

sensor_msgs/Image  depth

sensor_msgs/Image semantic

int32[]              poseId

geometry_msgs/Pose[] pose

int32              length

int32              loopclosureId

int32		    proximityDetectionId

sensor_msgs/PointCloud2 rtabmapPcl
